### DISCUSSION TITLE
容器名称的问题

### DISCUSSION THREAD

**Discussion by caisxg** [1Panel-dev/1Panel#4654] [Q&A]
我使用 docker ps 查看容器名称 `1Panel-mysql-yFjd`,  这是我的常识.

但是我准备把这个应用到其他容器的时候, 因为容器依赖 mysql 服务, 结果写 ip 的时候我用 `1Panel-mysql-yFjd` 失败

实战下来, 写 mysql 没问题?  请问容器名称到底是哪个? 有点混乱...

**Comment by wanghe-fit2cloud** [comment]
不同服务都使用相同的容器网络。
